src/components/JobSubtitle.tsx

```diff
@@ -0,0 +1,16 @@
+import { JobData } from "src/DataTypes";
+
+type Props = {
+    job: JobData
+};
+
+export default function JobSubtitle({ job }: Props) {
+    const hasSubtitle = job.location || job.salary || job.seniority;
+    return (hasSubtitle ? <span className="text-foreground-secondary">
+        (
+            {job.location || ''}
+            {job.salary ? ` / ${job.salary}` : ''}
+            {job.seniority ? ` / ${job.seniority}` : ''}
+        )
+    </span> : null)
```

Comment:
This is mostly a style choice, but, this can be written as:

```jsx
return hasSubtitle && (
    <span>[...]</span>
)
```

AFAIK, this is the case throughout?

Reply:
Not as the return statement of a component I don't think?

Reply:
Will try it.

Reply:
Yeah it doesn't seem to work.

Reply:
Seeing as though all three fields are required, there will always be a subtitle anyway, so I can remove that check.